modules/java-modules/elastic-v2/src/main/java/org/syslog_ng/elasticsearch_v2/ElasticSearchOptions.java

```diff
@@ -147,17 +148,21 @@ public boolean getSkipClusterHealthCheck() {
                 return options.get(SKIP_CLUSTER_HEALTH_CHECK).getValueAsBoolean();
         }
 
-	public String getClusterUrl() {
-		String cluster_url = options.get(CLUSTER_URL).getValue();
-		if (cluster_url.isEmpty()) {
-			StringBuilder url = new StringBuilder();
-			url.append("http://");
-			url.append(getServerList()[0]);
-			url.append(":");
-			url.append(getPort());
-			cluster_url = url.toString();
+	public Set<String> getClusterUrl() {
+		String[] cluster_url = options.get(CLUSTER_URL).getValueAsStringList(" ");
```

Comment:
maybe the name of the function should also be changed, getClusterUrls() would indicate it better that there are multiple values.

Also, is space the best separator in that option? Do we use that elsewhere? Just asking, not trying to suggest something.